style.css

```diff
@@ -1,3 +1,140 @@
-#root {
-  color: red;
+@import url("https://fonts.googleapis.com/css2?family=Baloo+Bhai+2&family=Open+Sans&display=swap");
+
+body {
+  margin: 0 auto;
+  background-color: #000;
+}
+header {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  align-content: center;
+  align-items: center;
+}
+
+@media (min-width: 876px) {
+  header {
+    flex-direction: row;
+    flex-wrap: nowrap;
+    background-color: #444444;
+    color: white;
+    background-image: linear-gradient(#000, #444444);
+  }
+}
+
+main {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
+  column-gap: 20px;
+  row-gap: 30px;
+  width: 98%;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 20px;
+}
+
+.episode,
+.shows {
+  background-color: #444444;
+  text-align: center;
+  color: white;
+}
+
+.episode:hover,
+.shows:hover {
+  cursor: pointer;
+  animation: scaleout 0.1s;
+  transform: scale(1.02, 1.02);
+}
+
+.episode img,
+.shows img {
+  width: 100%;
+}
+
+@keyframes scaleout {
+  20% {
+    transform: scale(1.01, 1.01);
+  }
+  60% {
+    transform: scale(1.02, 1.02);
+  }
+  100% {
+    transform: scale(1.03, 1.03);
+  }
+}
+
+.shows h1,
+.episode h1 {
+  font-family: "Baloo Bhai 2", cursive;
+  font-size: 1.25em;
+  border-top: 6px solid yellowgreen;
+  text-align: left;
+  padding-left: 5px;
+  margin-top: 0;
+  background-color: #444444;
+  color: white;
+  padding: 5px 20px;
+}
+
+.episode p,
+.shows p {
+  font-family: "Open Sans", sans-serif;
+  text-align: left;
+  padding: 4px 5px 20px 0;
+  margin-left: 15px;
+}
+
+.drdEpisodes {
+  background-color: #fff;
+  width: 300px;
+  height: 35px;
+  font-size: 1.125em;
+  margin: 10px;
+  padding: 5px;
+}
+
+.drdShows {
+  background-color: #fff;
+  width: 300px;
+  height: 35px;
+  font-size: 1.125em;
+  margin: 10px;
+  padding: 5px;
+}
+
+.search {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  flex-wrap: wrap;
+  padding: 15px;
+}
+
+.search input {
+  width: 300px;
```

Comment:
it is better to use a grid system if you want to give something width and height, remember lucy told us we are not allowed to use width and height together, it is better to use col and width or you can give your navbar or whatever your search is in there the width of 100% and just give the width of search like 60% of the parent or whatever you want